### ISSUE TITLE
What is the problem we are trying to solve

### ISSUE THREAD

**Issue by brianteeman** [joomla/jissues#203]
We need a single point of truth to manage issues, avoid repetition by posting in two places and ensure all comments are seen without following two places.
### Joomlacode

offers us a workflow for managing issues that is proven over time and works well for reporters and testers but less so for developers. See http://prezi.com/fp4bdgg2lprx/joomla-bug-squad-process/. It is easy to give an issue a status and to filter issues based on the status
### Github

offers us a great way to submit code and to review and comment directly on the code itself but no ability to record the status or categorise the issue
## The problems are as follows
1.  Having two different systems with no single point of truth
2.  Email notifications and tracking (github is great, joomlacode is bad)
3.  The need to create an account on two different systems
4.  Searching existing issues to see if it has already been reported (both are bad)
5.  Labelling an issue with a category (joomlacode is good, github has none)
6.  Recording the status of an issue (joomlacode is good, github has none)
   [note: github has labels but ONLY a maintainer can apply them]
7.  Ease of creating an issue and ensuring it is well documented
## Solving the problems

So what does jissues need to offer in order to completely replace joomlacode and are we there yet.
1. Github has a problem turning issues into pull requests and/or merging issues so we can end up with multiple issues for the same problem 
2. Email notifications and the links in them come from github not jissues
3. Accounts - you need to only use a github account BUT for a new user with no github account they are taken offsite to create this. WE should use the github api to create the account on our site (assuming this is possible)
4. Search works GREAT on jissues
5. There is no categorisation available in jissues eg templates, libraries, javascript
6. There is no status available (or i couldnt see it - ACL ?)
7. We have seen that feature requests created in github has much richer documentation and images etc than joomlacode which is awesome. I had hoped that the github experience would be present in jissues but currently it uses vanilla markdown with a pretty crummy toolbar. GFM (Github Flavoured Markdown) is much better and makes formatting and inserting images a breeze.
## TL:DR

jissues is a wrapper for github but doesnt remove the need for people to go to github to create a PR.
jissues doesnt offer the labelling and status fields that github lacks and joomlacode offers

jissues imho is only at version 0.50 ie it has 50% of the features/functionality that Joomla needs
